subsys/bluetooth/controller/ll_sw/nordic/lll/lll_peripheral_iso.c

```diff
@@ -613,13 +598,16 @@ static void isr_rx(void *param)
 			node_rx->hdr.type = NODE_RX_TYPE_ISO_PDU;
 			node_rx->hdr.handle = cis_lll->handle;
 			iso_meta = &node_rx->hdr.rx_iso_meta;
-			iso_meta->payload_number = (cis_lll->event_count *
-						    cis_lll->rx.bn) +
-						   (cis_lll->rx.bn_curr - 1U);
-			iso_meta->timestamp =
+			iso_meta->payload_number = cis_lll->rx.payload_count +
+						   cis_lll->rx.bn_curr - 1U;
+			iso_meta->timestamp = cis_lll->offset +
 				HAL_TICKER_TICKS_TO_US(radio_tmr_start_get()) +
-				radio_tmr_aa_restore() -
+				radio_tmr_aa_restore() - cis_offset_first -
```

Comment:
The changes here to timestamp comes from https://github.com/zephyrproject-rtos/zephyr/pull/60211, right?